src/lightly_train/_callbacks/export.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (c) Lightly AG and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+from __future__ import annotations
+
+from pathlib import Path
+
+from pytorch_lightning import LightningModule, Trainer
+from pytorch_lightning.callbacks import Callback
+from torch.nn import Module
+
+from lightly_train._commands import common_helpers
+from lightly_train._commands.common_helpers import ModelFormat
+from lightly_train._configs.config import PydanticConfig
+
+
+class ModelExportArgs(PydanticConfig):
+    every_n_epochs: int | None = None
```

Comment:
Shouldn't this be:
```suggestion
    every_n_epochs: int = 1
```

Then it is clear that it is exported after every epoch by default.

Reply:
yes, I only took inspiration from how it was done in checkpoint:    [every_n_epochs: int | None = None](https://github.com/lightly-ai/lightly-train/blob/ccc8ed10406918974ab2c96004207255f5531149/src/lightly_train/_callbacks/checkpoint.py#L33)

But I'm happy to change it, since it would also remove some logic.